### PR TITLE
War 1975: Copy settings apps' static files during server start up

### PR DESCRIPTION
**Pull request by bennfnc**
Katana handles automatic copy of all installed wapp static files into katana/static during server start up. 
Another ticket [ WAR-1968](https://github.com/warriorframework/warriorframework_py3/tree/WAR-1968) handles copy of static files during app install. 

Please review the changes and merge to develop.

### CODE REVIEW

**Comment by sanikakulkarni** [comment]
REST Regression is failing. Travis has passed otherwise

File: katana/utils/file_utils.py [hunk 1]
```diff
@@ -46,19 +47,94 @@ def readlines_from_file(path, start=None, end=None):
...
         else:
             output = False
             print("-- An Error Occurred -- {0}".format(e))
     return output
 
 
+def get_wapp_static(ref):
+    src_files = list_dir(ref)
+    file_list = []
+    for file in src_files:
```
Comment (sanikakulkarni) [inline katana/utils/file_utils.py]: ```file``` is a builtin type. Please use another variable name.

File: katana/wui/core/core_utils/core_utils.py [hunk 2]
```diff
@@ -97,3 +109,60 @@ def validate_config_json(json_data, warrior_dir):
         ordered_json["pythonpath"] = json_data["pythonpath"]
 
     return ordered_json
+
+
+def iscontainer():
+    """
+    iscontainer: When inside a container, control groups will match the pattern /docker/<containerid>
+    :return: True if control group points to docker or False if not inside a container
+    """
```
[the remark this answers is not in the document]
Comment (bennfnc) [reply katana/wui/core/core_utils/core_utils.py]: This was done to simulate a docker environment inside a vm. This isn't supposed to be part of the pull request. Also, there are multiple return statements. Let me rectify this.

File: katana/wui/core/core_utils/core_utils.py [hunk 3]
```diff
@@ -97,3 +109,60 @@ def validate_config_json(json_data, warrior_dir):
         ordered_json["pythonpath"] = json_data["pythonpath"]
 
     return ordered_json
+
+
+def iscontainer():
```
[the remark this answers is not in the document]
Comment (bennfnc) [reply katana/wui/core/core_utils/core_utils.py]: Yes, the function names should follow snake case convention.

**Comment by sanikakulkarni** [comment]
Closing PR as this functionality is not needed

**Pull request closed without merging**